bin/check-redis-memory-percentage.rb

```diff
@@ -80,8 +80,12 @@ def run
     redis = Redis.new(options)
 
     redis_info = redis.info
-    memory_in_use = redis_info.fetch('used_memory').to_f.div(1024).to_f             # used memory in KB (KiloBytes)
-    total_memory = redis_info.fetch('maxmemory', system_memory).to_f.div(1024).to_f # max memory (if specified) in KB
+    max_memory = redis_info.fetch('maxmemory', 0).to_f
```

Comment:
Makes sense, I am not a believer in keep existing logic because that is the way it was. I am a believer in trying to keep the same behavior if possible to avoid breaking change. Does redis return floats on # of kbs? If not then there is no value of casting it here and if division is needed to create a percentage then it can accommodate that there. It makes it clearer as to its intention. If redis actually returns a float I feel differently about this.